readthedocs/settings/dev/celery/builds.py

```diff
@@ -0,0 +1,27 @@
+"""Local setting for celery build workers (without DB access)"""
```

Comment:
Minor nit: `webs` / `builds` doesn't need to be plural. `webs`/`builds` is how we refer to the multiple servers in each grouping of servers internally, `web`/`build` would be how i would describe the celery grouping

Reply:
In fact, this is the wrong layout to use. readthedocsinc names the files with descending priority, so `readthedocsinc.settings.main.dev`, not `readthedocsinc.settings.dev.main`. This was important to account for, as readthedocsinc needs a comparable PR.